base_layer/wallet/src/transaction_service/protocols/transaction_receive_protocol.rs

```diff
@@ -435,7 +435,7 @@ where
             let completed_transaction = CompletedTransaction::new(
                 self.id,
                 self.source_address.clone(),
-                self.resources.wallet_identity.address.clone(),
+                self.resources.tari_address.clone(),
```

Comment:
Shoudl we define exactly which address this is for clarity. Instead of `tari_address` use a specific `intereactive_address`, `one_sided_address` etc. Or do we not know in this circumstance?

Reply:
good point, let me change this. These should all be interactive, so lets make it interactive.

Reply:
Going to fix this in a follow up pr